cscs-checks/mch/automatic_arrays_acc.py

```diff
@@ -12,10 +12,9 @@
 class AutomaticArraysCheck(rfm.RegressionTest):
     def __init__(self):
```

Comment:
Can you also adapt so that this uses class level attrributes instead of the older `__init__` method?